pkg/scheduler/scheduler.go

```diff
@@ -169,20 +168,17 @@ func (s *Scheduler) RegisterFromNodeAnnotations() {
 			nodeNames = append(nodeNames, val.Name)
 			for devhandsk, devInstance := range device.GetDevices() {
 				health, needUpdate := devInstance.CheckHealth(devhandsk, val)
+				klog.V(5).InfoS("node ", val.Name, "device", devhandsk, "health", health, "needUpdate", needUpdate)
 				if !health {
-					_, ok := s.nodes[val.Name]
+					err := devInstance.NodeCleanUp(val.Name)
```

Comment:
We should check ok this var, if the node does not exist, `devInstance.NodeCleanUp(val.Name)` should exec fail.